kie-maven-plugin/src/test/projects/kjar-2-all-resources/src/main/resources/AllResourceTypes/simple-dmn.dmn

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<definitions id="simple-dmn" name="simple-dmn"
+	namespace="https://github.com/droolsjbpm/kie-dmn"
+	xmlns="http://www.omg.org/spec/DMN/20151101/dmn.xsd"
+	xmlns:feel="http://www.omg.org/spec/FEEL/20140401">
+	<decision name="Greeting Message" id="d_GreetingMessage">
+		<variable name="Greeting Message" typeRef="feel:string"/>
+		<informationRequirement>
+			<requiredInput href="#i_FullName"/>
+		</informationRequirement>
+		<literalExpression>
+			<text>"Hello " + Full Name</text>
+		</literalExpression>
+	</decision>
+	<inputData name="Full Name" id="i_FullName">
+		<variable name="Full Name" typeRef="feel:string"/>
+	</inputData>

```

Comment:
:+1: seems my Eclipse is doing the TAB->SPACE substitution only for certain files.. updated indentation to suit.